config/detekt/detekt.yml

```diff
@@ -0,0 +1,9 @@
+formatting:
+  TrailingCommaOnCallSite:
+    active: true
+    autoCorrect: true
+    useTrailingCommaOnCallSite: true
+  TrailingCommaOnDeclarationSite:
+    active: true
+    autoCorrect: true
+    useTrailingCommaOnDeclarationSite: true
```

Comment:
Note that I only overwrote the relevant rules. It will keep using the default values for other rules.